product-service/src/main/java/com/extremecoder/productservice/dto/ErrorResponseDto.java

```diff
@@ -0,0 +1,17 @@
+package com.extremecoder.productservice.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
We are already using `@Builder`. So why we need `@AllArgsConstructor`?

Reply:
Resolved it. Thanks.